package.json

```
@@ -13,6 +13,8 @@
     "watch": "tsc -w",
     "lint": "eslint . --rulesdir utils/eslint-rules",
     "lint:fix": "npm run lint -- --fix",
+    "format": "prettier --check '**/*.ts'",
```

```suggestion
    "format:check": "prettier --check '**/*.ts'",
```

Makes intention a bit clearer IMO